server/src/main/java/org/spine3/server/stand/SubscriptionRegistry.java

```diff
@@ -68,7 +69,8 @@ synchronized void activate(Subscription subscription, Stand.EntityUpdateCallback
     synchronized Subscription addSubscription(Target target) {
         final String subscriptionId = Identifiers.newUuid();
         final String typeAsString = target.getType();
-        final TypeUrl type = TypeUrl.of(typeAsString);
+        final TypeUrl type = TypeName.of(typeAsString)
+                                     .toUrl();
```

Comment:
Is there any difference between `TypeName.of(smth).toUrl()` and `TypeUrl,parse(smth)`? If not, I suppose we should use the second option here.